website/src/app/kb/authenticate/oidc/readme.mdx

```diff
@@ -102,13 +98,25 @@ https://your-tenant.okta.com/.well-known/openid-configuration
 
 ### PKCE
 
-If the option is available, be sure to **enable PKCE** for the connector. This
-is a security feature that helps prevent certain types of attacks and is
-recommended for added security.
+The universal OIDC connector **requires** PKCE (Proof Key for Code Exchange) to
```

Comment:
Similar to the comment below, many people may not realize they are using the universal OIDC provider.  We may want to say something like "All providers using OIDC **require** PKCE...."

Reply:
True, but we know that the other ones _do_ support OIDC, so PKCE really only apply to the universal connector I think